frontend/widgets/OBSBasic_Transitions.cpp

```diff
@@ -1280,10 +1305,10 @@ void OBSBasic::EnableTransitionWidgets(bool enable)
 
 obs_data_array_t *OBSBasic::SaveTransitions()
 {
-	obs_data_array_t *transitions = obs_data_array_create();
+	obs_data_array_t *transitionsData = obs_data_array_create();
 
-	for (int i = 0; i < ui->transitions->count(); i++) {
-		OBSSource tr = ui->transitions->itemData(i).value<OBSSource>();
+	for (auto &pair : transitions) {
+		OBSSource tr = pair.second;
```

Comment:
```suggestion
	for (auto &[uuid, transition] : transitions) {
		OBSSource tr = pair.second;
```

I don't know if these could even be `const` as I don't expect anything within the loop body will pilfer or move them.

Reply:
Done